### PR TITLE
Thellimist/pi 914 Clean code

### PR DESCRIPTION
**Pull request by thellimist**

### CODE REVIEW

File: src/js/routes.js [hunk 1]
```diff
@@ -28,8 +26,6 @@ export default (
         <Route path={ UrlPaths.HOME_PAGE } component={ HomePage } onEnter={ requireAuth } />
         <Route path={ UrlPaths.ANALYTICS_PAGE } component={ AnalyticsPage } onEnter={ requireAuth } />
         <Route path={ UrlPaths.DOMAINS_OVERVIEW_PAGE } component={ DNSManagementPage } onEnter={ requireAuth } />
-        <Route path={ UrlPaths.PERFORMANCE_PAGE } component={ PerformancePage } onEnter={ requireAuth } />
-        <Route path={ UrlPaths.SECURITY_PAGE } component={ SecurityPage } onEnter={ requireAuth } />
```
Comment (jwineman) [inline src/js/routes.js]: We can remove these `UrlPaths` constants too right?

File: src/js/containers/SecurityPage/SecurityPage.js [hunk 2]
```diff
@@ -1,55 +0,0 @@
...
-        let { activeZoneId, zoneSettings } = this.props;
-        let isSettingsEmpty = _.isEmpty(zoneSettings.entities[activeZoneId]);
-
-        return (
-            <div>
-                {isSettingsEmpty && (<FormattedMessage id="errors.noActiveZoneSelected"/>)}
-                {!isSettingsEmpty && (
-                    <div>
-                        <Heading size={1}><FormattedMessage id="container.securityPage.title"/></Heading>
-                        <FeatureManager isEnabled={this.props.config.featureManagerIsSSLEnabled}>
```
Comment (jwineman) [inline src/js/containers/SecurityPage/SecurityPage.js]: Is anything else using `<FeatureManager/>`? Can we remove it too?

**Comment by jwineman** [comment]
LGTM assuming we still need `<FeatureManager/>`

File: src/js/containers/SecurityPage/SecurityPage.js (hunk 2 above)
Comment (thellimist) [inline src/js/containers/SecurityPage/SecurityPage.js]: `featureManagerIsFullZoneProvisioningEnabled` is still using `<FeatureManager/>`.

Comment (jwineman) [inline src/js/containers/SecurityPage/SecurityPage.js]: Eventually we can remove it and convert it to a selector.  Don't worry about it for now.

**Pull request merged**